packages/amplify-e2e-tests/package.json

```diff
@@ -49,12 +49,15 @@
     "execa": "^5.1.1",
     "extract-zip": "^2.0.1",
     "fs-extra": "^8.1.0",
+    "get-port": "^5.1.1",
     "glob": "^8.0.3",
     "graphql-tag": "^2.10.1",
     "graphql-transformer-core": "^8.1.9",
     "isomorphic-fetch": "^3.0.0",
     "lodash": "^4.17.21",
+    "moment": "^2.24.0",
```

Comment:
is Github not showing the correct change or we are adding new packages in this PR?

Reply:
We're not adding new packages.
This is being used here https://github.com/aws-amplify/amplify-cli/blob/015bc38d490d265be19c0686769449dbf0e8313a/packages/amplify-e2e-tests/src/__tests__/transformer-migrations/auth-migration.test.ts#L24 .

But was not added as explicit dependency.
It was working because other packages pulled it and `yarn` kept common link between node_modules.

`import/no-extraneous-dependencies` is catching this kind of abuse and forces explicit dependencies.

Reply:
More about the rule https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-extraneous-dependencies.md

Reply:
And see https://github.com/search?q=repo%3Aaws-amplify%2Famplify-cli%20%27moment%27&type=code for where this package is also used in the repo.